<?xml version="1.0"?>
<launch>
  <rosparam command="load" file="$(find joy_interactive_transform_publisher)/config/joy_keymap.yaml"/>

  <node pkg="joy_interactive_transform_publisher" type="joy_interactive_transform_publisher" name="joy_interactive_transform_publisher" output="screen">
    <param name="world_frame" value="world"/>
    <param name="child_frame" value="joy_frame"/>
    <param name="yaw_scale" value="0.01"/>
    <param name="pitch_scale" value="0.01"/>
  </node>

</launch>